packages/api/src/routes/blog-subscribe.js

```diff
@@ -0,0 +1,81 @@
+import { JSONResponse } from '../utils/json-response.js'
+
+const SERVER_PREFIX = 'us5'
+const LIST_ID = '64f6e3fd11'
+const API_KEY = MAILCHIMP_API_KEY
+const TOKEN = Buffer.from(`:${API_KEY}`).toString('base64')
```

Comment:
As mentioned in another comment, we should avoid Node.js specific APIs as we're not running in a Node.js environment.

> I can't see any indication that [`btoa`](https://developer.mozilla.org/en-US/docs/Web/API/btoa) is deprecated on MDN.
>
> Personally I'd reach for this over pulling in a Node.js specific library to the bundle and suppress the warning instead.

Reply:
i went down a rabbithole of getting warnings/errors telling me multiple things were deprecated or to use other things until settling on this. I'll just use btoa and ignore the warnings